workflow/common/common.go

```diff
@@ -34,6 +34,10 @@ const (
 
 	// AnnotationKeyNodeName is the pod metadata annotation key containing the workflow node name
 	AnnotationKeyNodeName = workflow.FullName + "/node-name"
+
+	// AnnotationIncludeOutputs is the pod metadata annotation key containing flag to include script output
+	AnnotationIncludeOutputs = workflow.FullName + "/include-output"
```

Comment:
Instead of a new annotation, we have an existing datastructure, ExecutionControl, which we should add a field to to control this.